go.mod

```diff
@@ -5,7 +5,7 @@ go 1.24.0
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/gardener/gardener v1.126.0
-	github.com/gardener/gardener/hack/tools/logcheck v0.0.0-20250729135036-f5df05714b06
+	github.com/gardener/gardener/hack/tools/logcheck v0.0.0-20250823000107-854d2916d618
```

Comment:
Shouldn't `github.com/gardener/gardener/hack/tools/logcheck` revision correspond to the `github.com/gardener/gardener` version? The commit sha behind `github.com/gardener/gardener@v1.126.0` is https://github.com/gardener/gardener/commit/1f7741570c311a5a2bb369a7a242d896a1cea496, not - https://github.com/gardener/gardener/commit/854d2916d618.